specs/networking/p2p-interface.md

```diff
@@ -160,13 +178,15 @@ Additional topics are used to propagate lower frequency validator messages. Thei
 
 #### Interop
 
-Unaggregated and aggregated attestations from all shards are sent to the `beacon_attestation` topic. Clients are not required to publish aggregate attestations but must be able to process them. All validating clients SHOULD try to perform local attestation aggregation to prepare for block proposing.
+Unaggregated and aggregated attestations from all shards are sent as `Attestation`s to the `beacon_aggregate_and_proof` topic. Clients are not required to publish aggregate attestations but must be able to process them. All validating clients SHOULD try to perform local attestation aggregation to prepare for block proposing.
```

Comment:
I'm a bit torn here. Think we should leave the interop spec as it was. I can add back in the `beacon_attestation` channel for global propagation of individual attestations and we can just deprecate it for mainnet configurations

Reply:
I think I like the `ATTESTATION_SUBNET_COUNT == 1` since it is close to mainnet, but super simple as there is only one subnet (so not really a subnet, can be implemented easily by everone). Let's ask on the call maybe?

Reply:
thoughts on that?

Reply:
I suspect we should do an `interopv2` which is basically mainnet-parameters but testnet - this way we can leave interop as is for now and let people get the basics right  (we need this for noise and the other mainnet changes anyway)

it looks like it should possible that clients that support aggregation can start publishing to the aggregation topics for testing as soon as they're ready.

Reply:
I added back in the `beacon_attestation` topic for interop with a note that it will be removed for mainnet. Agreed that this simple procedure should stick around a little longer